mandatory/2-function-creation.js

```diff
@@ -15,26 +17,59 @@ function isAcceptableUser(userAge, isLoggedIn) {}
     is applieds and 142.5 should be returned)
 */
 
-function applyDiscount(totalPrice) {}
+function applyDiscount(totalPrice) {
+  if (totalPrice > 200) {
+    totalPrice -= (totalPrice * 10) / 100;
+    return totalPrice;
+  } else if (totalPrice < 200) {
+    totalPrice -= (totalPrice * 5) / 100;
+    return totalPrice;
+  }
+}
 
 /*
   Complete the function to print to the console the odd numbers between 1 and limit (use a while loop):
   */
-function printOddNumbers(limit) {}
+function printOddNumbers(limit) {
+  let x = 1;
+  while (x <= limit) {
+    if (x % 2 != 0) {
```

Comment:
This works perfectly. 
If you wanted to avoid using a negative (not !) condition, you could have said if 
(x % 2 == 1) because we are only dealing with two possible values of 0 or 1?